test/Validate.CFG.cpp

```diff
@@ -638,7 +639,8 @@ TEST_F(ValidateCFG, NestedLoops) {
   str += loop1 >> vector<Block>({loop2, loop1_merge});
   str += loop2 >> vector<Block>({loop2, loop2_merge});
   str += loop2_merge >> loop1_merge;
-  str += loop1_merge;
+  str += loop1_merge >> vector<Block>({loop1, exit});
```

Comment:
Let me see if I got this right: loop2 is loop1's continue target, and loop1_merge is loop1's back-edge block?  But loop2 doesn't dominate loop1_merge (there's a path entry >> loop1 >> loop1_merge), violating the rule that the continue target must dominate the back-edge block.